app/src/main/java/uk/ac/ebi/atlas/experimentpage/tabs/ExperimentPageContentService.java

```diff
@@ -125,19 +126,28 @@ public JsonArray getDownloads(String experimentAccession, String accessKey) {
                 ImmutableList.of(
                         ExperimentFileType.EXPERIMENT_METADATA,
                         ExperimentFileType.EXPERIMENT_DESIGN);
-
-        var resultFiles = isSmartExperiment(technologyType) ?
-                ImmutableList.of(
-                        ExperimentFileType.CLUSTERING,
-                        ExperimentFileType.QUANTIFICATION_FILTERED,
-                        ExperimentFileType.MARKER_GENES,
-                        ExperimentFileType.NORMALISED,
-                        ExperimentFileType.QUANTIFICATION_RAW) :
-                ImmutableList.of(
-                        ExperimentFileType.CLUSTERING,
-                        ExperimentFileType.MARKER_GENES,
-                        ExperimentFileType.NORMALISED,
-                        ExperimentFileType.QUANTIFICATION_RAW);
+        List<ExperimentFileType> resultFiles = new ArrayList<>();
+
+        if (experimentAccession.startsWith("E-ANND-")) {
+            // Mandatory files
+            resultFiles.add(ExperimentFileType.MARKER_GENES);
+            resultFiles.add(ExperimentFileType.NORMALISED);
+            // Optional CLUSTERING
+            if (dataFileHub.getSingleCellExperimentFiles(experimentAccession).getClustersTsv().exists() &&
```

Comment:
Could you please extract this long conditional check to a well named private method?
Thanks